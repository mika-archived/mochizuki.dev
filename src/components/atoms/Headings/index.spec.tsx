import "@testing-library/jest-dom/extend-expect";
import "jest-styled-components";

import React from "react";
import { cleanup, render } from "@testing-library/react";

import { Heading2 } from ".";

afterEach(cleanup);

test("does not have changes from snapshot in Heading2", () => {
  const { asFragment } = render(<Heading2>Heading 2</Heading2>);

  expect(asFragment()).toMatchSnapshot();
});
