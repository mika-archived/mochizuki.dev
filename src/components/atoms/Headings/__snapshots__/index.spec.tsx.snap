// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does not have changes from snapshot in Heading2 1`] = `
<DocumentFragment>
  .c0 {
  font-family: "游ゴシック体","Yu Gothic",YuGothic,"ヒラギノ角ゴシック Pro","Hiragino Kaku Gothic Pro",メイリオ,Meiryo,sans-serif;
  font-size: 21px;
  font-weight: normal;
}

<h2
    class="c0"
  >
    Heading 2
  </h2>
</DocumentFragment>
`;
